content/docs/started/getting-started-gke.md

```diff
@@ -23,6 +23,10 @@ brings the following advantages:
   * [Cloud Identity-Aware Proxy (Cloud IAP)](https://cloud.google.com/iap/) 
     makes it easy to securely connect to Jupyter and other
     web apps running as part of Kubeflow.
+  * Basic auth service supports simple username/password access to your 
+    Kubeflow. It is an alternative to Cloud IAP service. We recommend IAP for 
+    production and enterprise workloads. Consider basic auth **ONLY** when 
+    trying to test out Kubeflow and use it **WITHOUT** sensitive data.
   * [Stackdriver](https://cloud.google.com/logging/docs/) makes it easy to 
```

Comment:
Add a comment in the OAuth  client setup section that users can skip it if they are using basic auth.

Reply:
added.